src/app/_utils/categoryUtils.ts

```diff
@@ -43,10 +43,8 @@ export function getCategoryDataFromDirectory(directoryPath: string) {
   return categoryMap
 }
 
-export function getCategorySettings(
-  CMSCollectionKey: keyof CMSCategoryFieldMapping,
-) {
-  const rawCategories = getCategoryData(CMSCollectionKey)
+export function getCategorySettings(collectionName: CMSCollectionName) {
```

Comment:
Yes, I agree. Let's pick this refactor up in a different PR together. Since it requires a bit more thinking than just changing the name of this function. Considering the following types where `CategoryOption` is the `id` of the `CategorySetting`:

```
export type CategoryOption = string
export type CategoryMap = Record<string, string>
export type CategoryCounts = Record<CategoryOption, number>

export type CategorySetting = {
  id: CategoryOption
  name: string
}

export type CategoryYAMLData = {
  name: string
  slug: string
  subcategories: string[]
}

type CMSCategoryField = {
  collection: string
  field: string
}

export type CMSCategoryFieldMapping = {
  blog: CMSCategoryField
  ecosystem: CMSCategoryField
  events: CMSCategoryField
}
```